tests/nightly/JenkinsfileForBinaries

```diff
@@ -106,6 +106,14 @@ core_logic: {
           utils.docker_run('ubuntu_nightly_gpu', 'nightly_tutorial_test_ubuntu_python3_gpu', true)
         }
       }
+    },
+    'Clojure Nightly: CPU': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/clojure') {
+          utils.init_git()
```

Comment:
Please have a look at the other pipelines. You are missing the unpacking here. This is necessary because MXNet is compiled in a previous step and otherwise, this job has no artifact to run on. Also note that we currently only compile GPU for nightly.

Reply:
hmmm .. I don't see any other JVM nightlies to rely on. 
How can the compiled MxNet be connected to the Clojure code ?